lib/block_view_dao_coin_limit_order_test.go

```diff
@@ -1732,6 +1731,356 @@ func TestDAOCoinLimitOrder(t *testing.T) {
+		require.NoError(err)
+		require.NotEmpty(utxoEntriesM2)
+
+		txnMeta.BidderInputs = append(
+			[]*DeSoInputsByTransactor{},
+			&DeSoInputsByTransactor{
+				TransactorPublicKey: NewPublicKey(m2PkBytes),
+				Inputs:              append([]*DeSoInput{}, (*DeSoInput)(utxoEntriesM2[0].UtxoKey)),
+			})
+
+		_, _, _, _, err = _connectDAOCoinLimitOrderTxn(
+			testMeta, m1Pub, m1Priv, currentTxn, totalInputMake)
+		require.Error(err)
+		require.Contains(err.Error(), RuleErrorDAOCoinLimitOrderOverspendingDESO)
+
+		// m1 swaps out m0's BidderInputs for spent UTXOs
+		// from m0 and tries to connect. Should error.
+		utxoEntriesM0, err := chain.GetSpendableUtxosForPublicKey(m0PkBytes, mempool, nil)
+		require.NoError(err)
+		require.NotEmpty(utxoEntriesM0) // Unspent UTXOs exist for m0.
+
+		// Spend m0's existing UTXO.
+		tempUtxoView, err := NewUtxoView(db, params, chain.postgres)
+		require.NoError(err)
+		utxoOp, err := tempUtxoView._spendUtxo(utxoEntriesM0[0].UtxoKey)
+		require.NoError(err)
+		err = tempUtxoView.FlushToDb()
+		require.NoError(err)
+		utxoEntriesM0, err = chain.GetSpendableUtxosForPublicKey(m0PkBytes, mempool, nil)
+		require.NoError(err)
+		require.Empty(utxoEntriesM0) // No unspent UTXOs exist for m0.
+
+		txnMeta.BidderInputs = append(
+			[]*DeSoInputsByTransactor{},
+			&DeSoInputsByTransactor{
+				TransactorPublicKey: NewPublicKey(m0PkBytes),
+				Inputs:              append([]*DeSoInput{}, (*DeSoInput)(utxoOp.Key)),
+			})
+
+		_, _, _, _, err = _connectDAOCoinLimitOrderTxn(
+			testMeta, m1Pub, m1Priv, currentTxn, totalInputMake)
+		require.Error(err)
+		require.Contains(err.Error(), RuleErrorDAOCoinLimitOrderBidderInputNoLongerExists)
+
+		// Unspend m0's existing UTXO.
+		err = tempUtxoView._unSpendUtxo(utxoOp.Entry)
+		require.NoError(err)
+		err = tempUtxoView.FlushToDb()
+		require.NoError(err)
+		utxoEntriesM0, err = chain.GetSpendableUtxosForPublicKey(m0PkBytes, mempool, nil)
+		require.NoError(err)
+		require.NotEmpty(utxoEntriesM0) // Unspent UTXOs exist for m0.
+
+		// m1 includes m0's BidderInputs in addition to
+		// their own and tries to connect. Should error.
+		bidderInputs := append([]*DeSoInputsByTransactor{}, originalBidderInput)
+
+		bidderInputs = append(
+			bidderInputs,
+			&DeSoInputsByTransactor{
+				TransactorPublicKey: NewPublicKey(m1PkBytes),
+				Inputs:              append([]*DeSoInput{}, (*DeSoInput)(utxoEntriesM1[0].UtxoKey)),
+			})
+
+		txnMeta.BidderInputs = bidderInputs
+
+		_, _, _, _, err = _connectDAOCoinLimitOrderTxn(
+			testMeta, m1Pub, m1Priv, currentTxn, totalInputMake)
+		require.Error(err)
+		require.Contains(err.Error(), RuleErrorDAOCoinLimitOrderFeeNanosBelowMinTxFee)
+
+		// m1 includes m0's BidderInputs in addition to
+		// m2's and tries to connect. Should error.
+		bidderInputs = append([]*DeSoInputsByTransactor{}, originalBidderInput)
+
+		bidderInputs = append(
+			bidderInputs,
+			&DeSoInputsByTransactor{
+				TransactorPublicKey: NewPublicKey(m1PkBytes),
```

Comment:
```suggestion
				TransactorPublicKey: NewPublicKey(m2PkBytes),
```

Reply:
We can optionally make this change. Either way, should result in the same error.

Reply:
Made the change below.